CONTRIBUTING.md

```diff
@@ -41,6 +42,21 @@ ni
 nr dev
 ```
 
+#### Troubleshooting
+
+If you encounter any issues with the installation, we suggest trying the following steps:
+
+- Reinstall the dependencies:
+
+1. Remove the `node_modules` folder in Nimbus's root folder
+2. Retry the installation with `pnpm i` or `ni` if you have it installed
```

Comment:
pnpm lock file changes should be reverted and run `pnpm install --frozen-lockfile` or `ni --frozen`, in fact we should add this command after cloning repo to local.

pnpm lock file shouldn't be changed if not adding/updating dependencies

Reply:
Noted, I'll update all the installation instructions to add the frozen flag then.

Reply:
> pnpm lock file changes should be reverted and run `pnpm install --frozen-lockfile` or `ni --frozen`, in fact we should add this command after cloning repo to local.

In fact we should add this command after cloning repo to local and before creating a new branch.

Reply:
> In fact we should add this command after cloning repo to local and before creating a new branch.

I'm not sure I follow there, is there another place in the documentation I have missed where you would want to add this change ?

Reply:
If adding some dependency in the new PR, just add it using `ni add <dep> -w` or `ni add -D <dep> -w` or using pnpm `pnpm add <dep> -w` or `pnpm add -D <dep> -w`, will require `-w` flag since we're using pnpm monorepo (or add it manually in the corresponding `package.json` file and then run `pnpm install` or `ni`)

Reply:
Alright, I added a new paragraph In `CONTRIBUTING.md` to mention what you explained here.

Reply:
can you add that `ni` or `pnpm install`  should be run from root folder? iirc pnpm should work also if we run the command from docs folder, (maybe will show show warning/error), but it would be better to run it always from root folder

Reply:
It's done. 😊